cli/bin/grain.js

```diff
@@ -140,28 +140,32 @@ program
   // The root command that compiles & runs
   .arguments("<file>")
   .action(function (file) {
-    actions.run(actions.compile(file, program), program.opts());
+    var normalised_file = file.replace(/\\/g, "/");
+    actions.run(actions.compile(normalised_file, program), program.opts());
```

Comment:
We should probably have a linter, but we can change this slightly to match the file's code style:
```suggestion
    const normalisedFile = file.replace(/\\/g, "/");
    actions.run(actions.compile(normalisedFile, program), program.opts());
```